test/conformance/tests/sriov_operator.go

```diff
@@ -938,6 +938,66 @@ var _ = Describe("[sriov] operator", func() {
 				createUnschedulableTestPod(testNode, []string{sriovNetworkName}, resourceName)
 			})
 		})
+
+		Context("SRIOV and macvlan", func() {
+			// 25834
+
+			var macvlanNad netattdefv1.NetworkAttachmentDefinition
+
+			AfterEach(func() {
+				if &macvlanNad != nil {
+					clients.Delete(context.Background(), &macvlanNad)
+				}
+			})
+
+			It("Should be able to create a pod with both sriov and macvlan interfaces", func() {
+
+				resourceName := "sriovnic"
+				sriovNetworkName := "sriovnetwork"
+				testNode := sriovInfos.Nodes[0]
+
+				sriovDevice, err := sriovInfos.FindOneSriovDevice(testNode)
+
+				Expect(err).ToNot(HaveOccurred())
+				createSriovPolicy(sriovDevice.Name, testNode, 5, resourceName)
+
+				ipam := `{"type": "host-local","ranges": [[{"subnet": "1.1.1.0/24"}]],"dataDir": "/run/my-orchestrator/container-ipam-state"}`
```

Comment:
Can you please add here a range
```
"rangeStart": "10.1.1.10",
"rangeEnd": "10.1.1.20",
```

In a follow PR I will like to see a connection testing between the VF and the macvlan interface